analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -462,8 +462,8 @@ open class Npm(
         val json = jsonMapper.readTree(packageJsonFile)
 
         return ModuleInfo(
-            name = json["name"].textValue(),
-            version = json["version"].textValue(),
+            name = json["name"].textValueOrEmpty(),
```

Comment:
ORT does report an issue if there multiple projects with the same ID.
This change IMO makes that uniqueness slightly less clear.
I fear we should add a fallback mechanism to derive a more unique name.

Would you agree that we should use the path relative to the analyzer root as in PIP? @mnonnenmacher @sschuberth what do you think?

Reply:
That would make more sense to me than using an empty string for the name.

Reply:
I basically agree, but would like to tracks and tackle this separately. I just created https://github.com/oss-review-toolkit/ort/issues/3159 as a reminder for it.

Reply:
I'm ok with that.